src/microbit/model/microbit.py

```diff
@@ -0,0 +1,15 @@
+from .button import Button
+
+
+class MicrobitModel:
+    def __init__(self):
+        # State in the Python process
+        self.button_a = Button()
+        self.button_b = Button()
+
+    # Will be removed once functions added to shim
+    def show_message(self, message):
+        print("message: " + message)

```

Comment:
nit: should have To be implemented, also it seems this function is already called in the shim

Reply:
done, added real functions that are part of microbit library